base/linalg/triangular.jl

```diff
@@ -1087,15 +1174,116 @@ function logm{T<:Union{Float64,Complex{Float64}}}(A0::UpperTriangular{T})
          1.276404810806775e-001,
          2.060962623452836e-001,
          2.879093714241194e-001]
+
+    n = chksquare(A0)
+
+    A, m, s = invsquaring(A0,theta)
+
+    # Compute accurate diagonal of A - I
+    sqrt_diag!(A0, A, s)
+
+    # Compute the Gauss-Legendre quadrature formula
+    R = zeros(Float64, m, m)
+    for i = 1:m - 1
+        R[i,i+1] = i / sqrt((2 * i)^2 - 1)
+        R[i+1,i] = R[i,i+1]
+    end
+    x,V = eig(R)
```

Comment:
(If you were to do this here, you would probably want to use a `SymTridiagonal` matrix `R`, no?)